test/Conversion/tritongpu_to_llvm_blackwell.mlir

```diff
@@ -386,10 +386,10 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32} {
 
 module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32, ttg.shared = 65544 : i32, ttg.target = "cuda:100", ttg.tensor_memory_size = 128 : i32, "ttg.threads-per-warp" = 32 : i32} {
   // CHECK-LABEL: @tensor_memory_ld_128x256
-  // CHECK-COUNT-4: tcgen05.st.sync.aligned.32x32b.x64.b32
+  // CHECK-COUNT-2: tcgen05.st.sync.aligned.32x32b.x128.b32
   // CHECK-NOT: tcgen05.st
   // CHECK: nvvm.tcgen05.wait <store>
-  // CHECK-COUNT-4: tcgen05.ld.sync.aligned.32x32b.x64.b32
+  // CHECK-COUNT-2: tcgen05.ld.sync.aligned.32x32b.x128.b32
```

Comment:
This test case checks that the special-case x128->x64 heuristic is working. This should definitely be using vec=64, not 128